### ISSUE TITLE
How to continue training a LoRA

### ISSUE THREAD

**Issue by Imusing** [Bikecicle/LoRAW#13]
I trained a LoRA, stopped training, and I want to continue training from the same one. But using --lora-ckpt-path <lorapath> just errors with this traceback:

```
File "D:\tts\ok\stable-audio-tools-main\train.py", line 78, in main
    torch.load(args.lora_ckpt_path, map_location="cpu")["state_dict"]
KeyError: 'state_dict'
```

**Comment by Imusing** [comment]
Ok, removing ["state_dict"] works (currently testing if training works).

**Comment by Imusing** [comment]
I now get a different error. Might be because I changed the dataset a little.

**Comment by NeuralNotW0rk** [comment]
I can investigate the issue in more depth much later, but for now what is the new error you are getting?

**Comment by Imusing** [comment]
> I can investigate the issue in more depth much later, but for now what is the new error you are getting?

```File "D:\tts\ok\stable-audio-tools-main\loraw\modules.py", line 71, in forward
     norm_scale = self.dora_mag.weight.view(-1) / (torch.linalg.norm(new_weight_v, dim=1)).detach()
 RuntimeError: The size of tensor a (3072) must match the size of tensor b (1536) at non-singleton dimension 0
```

That is the new error

**Comment by Imusing** [comment]
Nope, even when not changing the dataset between 2 runs, it still gives that error

**Comment by Gab404** [comment]
Did you fix your error ? Because I have the same issue

**Comment by Imusing** [comment]
> Did you fix your error ? Because I have the same issue

No, I just kept retraining the lora from scratch.

**Comment by Gab404** [comment]
I fixed it on my side. I just removed `state_dict` and loaded the weights on the gpu. 

Run with : `python3 ./train.py --dataset-config /path/to/dataset_config.json --model-config ./stable_audio_tools/configs/model_configs/txt2audio/stable_audio_1_0.json --lora-ckpt-path /path/to/lora.ckpt --pretrained-ckpt-path /path/to/model.ckpt --name lora_fine_tuning --save-dir ./ --use-lora true`

Working on Google Colab A100
